benchmarks/online-shop/productcatalogservice/go.mod

```diff
@@ -3,43 +3,37 @@ module github.com/GoogleCloudPlatform/microservices-demo/src/productcatalogservi
 go 1.18
 
 require (
```

Comment:
@dhschall why do we have 2 `require(...)` statements?

Reply:
Oh these are go 1.18. I think from go 1.17 it became that the first `require` statement would have direct dependencies and the second would only have transitive dependencies.

Reply:
hmm... ok maybe we should consider going to 1.18 after all.... @dhschall what do you think?

Reply:
Yes, we should. vHive is 1.18
I already started to move some of my stuff as I run into a version problem when I wanted to install vHive